### PR TITLE
tests(blocks): fix upload and data export cubit tests

### PR DESCRIPTION
**Pull request by karlprieb**
Changes from #665

### CODE REVIEW

File: test/test_utils/fake_data.dart [hunk 1]
```diff
@@ -14,6 +12,6 @@ const testEntityName = 'test-name';
 const testEntitySize = 10;
 const testEntityDataContentType = 'test-data-mime-type';
 const testEntityDataTxId = 'FxgiSM3JOAsjnZF-Yzv2z7h-Io4K6GmCarhQ0qAUL6I';
-const testEntityId = Uuid().v4();
-const driveId = Uuid().v4();
-const rootFolderId = Uuid().v4();
+const testEntityId = 'c1dda6a2-0d74-4c35-9671-54e450a778fd';
```
Comment (agsuy) [inline test/test_utils/fake_data.dart:15]: I do not think removing random `Uuid().v4()` provides us better value.

When to use random generation vs. when to use constants usually comes down to what I am testing and if I care about the function generating those values or not. (e.g. if we use that random function only in tests or its a function used by our codebase to perform certain tasks.)

In this case, we use `Uuid().v4()` in our upload plan, so we definitely care about that function hence we should keep it.

**Review by agsuy — CHANGES_REQUESTED** [review]
Please keep `Uuid().v4()`

File: test/blocs/upload_cubit_test.dart [hunk 2]
```diff
@@ -58,10 +55,10 @@ void main() {
     registerFallbackValue(Wallet());
     registerFallbackValue(FolderEntry(
         id: '',
-        dateCreated: DateTime.now(),
+        dateCreated: tDefaultDate,
```
[the remark this answers is not in the document]
Comment (karlprieb) [reply test/blocs/upload_cubit_test.dart:58]: dateCreated doesn't matter here, so we're changing to calculate it just once.

File: test/test_utils/fake_data.dart (hunk 1 above)
Comment (karlprieb) [reply test/test_utils/fake_data.dart]: agreed!
[bede10e](https://github.com/ardriveapp/ardrive-web/pull/678/commits/bede10e9a857626319c0f94ffbc898795c29ce74)

**Pull request merged**